### PR TITLE
ADEN-1754 Remove invisible slots

### PR DESCRIPTION
**Pull request by nandy-andy**
https://wikia-inc.atlassian.net/browse/ADEN-1754

I've removed them from everywhere except `AdEngine/schema.sql`.

### CODE REVIEW

File: extensions/wikia/AdEngine/js/AdProviderLiftium.js [hunk 1]
```diff
@@ -22,8 +22,6 @@ define('ext.wikia.adEngine.provider.liftium', [
 		'HOME_TOP_LEADERBOARD': {'size': '728x90'},
 		'HOME_TOP_RIGHT_BOXAD': {'size': '300x250'},
 		'INCONTENT_BOXAD_1': {'size': '300x250'},
-		'INVISIBLE_1': {'size': '0x0', 'useGw': true},
-		'INVISIBLE_2': {'size': '0x0', 'useGw': true},
```
Comment (gabrys) [inline extensions/wikia/AdEngine/js/AdProviderLiftium.js]: Please also clean up the if (useGw) part and I think the dependency on wikia.scriptWriter

Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/AdProviderLiftium.js]: I'm fixing it! :sweat_smile:

Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/AdProviderLiftium.js]: Done :smile_cat:

**Comment by gabrys** [comment]
LGTM

**Pull request merged**